### ISSUE TITLE
HTML reports WCAG version

### ISSUE THREAD

**Issue by DayanaFigueredo** [microsoft/accessibility-insights-web#3551]
The report might bring some concerns to the final reviewers because the WCAG Criteria version has not been updated.

When every new WCAG Criteria version is released report reviewers would request feedback from the latest version. So if the criteria is supported on the Accessibility Insigths tool, this should be reflected on the report as well.

The supported WCAG version should be displayed at all times on the extension and while using the features, so users do not get confused and think that the criteria has not been exercised properly at the time of exporting the report.

I installed and worked with this tool today. At the moment the WCAG 2.1 is the latest version available, and my report reflects the previous version instead.

<img width="1021" alt="Screen Shot 2020-10-28 at 4 02 54 PM" src="https://user-images.githubusercontent.com/46756658/97496552-16b83a00-1937-11eb-9a46-7bfe0032c473.png">

**Comment by ferBonnin** [comment]
Hi @DayanaFigueredo this is now fixed in the [Canary version](https://chrome.google.com/webstore/detail/hbcplehnakffdldhldncjlnbpfgogbem) of Accessibility Insights. Please take a look and let us know what you think!
